### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by okoyecharles**
## Description ✏️
This is the last project for the Bookstore website. Now that all the functionality is done.

## Changes Required ♻️
- [ ]  Use the information detailed in Zeplin to style the website.
- [ ] Fix linter errors.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -18,19 +18,23 @@ function Book({ id, title, author }) {
...
       <div className="book__progress">
-        <div className="book__progressCircle">Ø</div>
-        <div className="book__progressPercent">64%</div>
-        <div className="book__progressStatus">Completed</div>
+        <div className="book__progressCircle">o</div>
+        <div>
+          <div className="book__progressPercent">64%</div>
+          <div className="book__progressStatus">Completed</div>
+        </div>
       </div>
```
Comment (Whoistolu) [inline src/components/Book.js:34]: - [ ] [REQUIRED] Great design you have here @OkoyeCharles , however, you need to ensure that you stick to the color pattern provided in the  [zeplin template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). By fixing this, you will be sticking to the project requirement.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
 |

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @OkoyeCharles 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 1 above)
Comment (okoyecharles) [reply src/components/Book.js:34]: Thanks very much for the review and I will implement that immediately. Charles

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @OkoyeCharles,

Good job on implementing the changes asked by the previous reviewer 👍

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly
✅ All linter checks passed
✅ App is nicely designed as expected 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**